### ISSUE TITLE
No audio on Rpi4 with Raspberry Pi OS x64

### ISSUE THREAD

**Issue by zaisty** [runelite/runelite#17145]
### Type

Incorrect behavior

### Operating System

Linux

### OS Version

Raspberry Pi OS x64

### Bug description

There's no sound at all with latest arm64 AppImage on Raspbbery Pi Os x64.
I try openjdk 17 (default from repo) and binary openjdk 21.0.1.
Nothing helps.

### RuneLite version

RuneLite arm64 AppImage Latest

**Comment by laughingbiscuit** [comment]
it audio working on other applications? do you have the line `dtparam=audio=on` in `/boot/config.txt`?

**Comment by zaisty** [comment]
Yes I have dtparam=audio=on in /boot/config.txt, and ALL other applications/games works without problems with audio/sound.

**Comment by zaisty** [comment]
OK now it works! Uff!
This fix I mention its WORKING!

[Here](http://myrobotlab.org/content/fixing-audio-output-java-11-raspberry-pi) is complete solution to how to make audio wokrs with java openjdk.

Just edit sound.properties and add this lines to it:
`javax.sound.sampled.Clip=com.sun.media.sound.DirectAudioDeviceProvider

javax.sound.sampled.Port=com.sun.media.sound.PortMixerProvider

javax.sound.sampled.SourceDataLine=com.sun.media.sound.DirectAudioDeviceProvider

javax.sound.sampled.TargetDataLine=com.sun.media.sound.DirectAudioDeviceProvider`

But YOU MUST restart device.
I don't do that and it dont work.

Also I switch to RuneLite.jar instead of AppImage.
Finally everything is working.
